bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/discovery/ZonePlayerDiscoveryParticipant.java

```diff
@@ -95,6 +95,9 @@ public Set<ThingTypeUID> getSupportedThingTypeUIDs() {
                     case "Arc SL":
                         modelName = "ArcSL";
                         break;
+                    case "Roam SL":
+                        modelName = "RoamSL";
```

Comment:
Perhaps it would make sense as `default` either to strip invalid characters or set `ignored = true` if such characters are present. Otherwise we'll get `IllegalArgumentException` anyway when trying to create the ThingTypeUID. This could also be logged. What do you think?

Reply:
As reading PR's in sequence I hadn't seen #12609 when writing this comment. Please disregard. :)